.github/workflows/test_integration_jaas.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
   test:
     name: Integration-JAAS
     needs: build
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, jammy, x64]
```

Comment:
Ah I remember I investigated moving to self hosted runners and it didn't work because docker compose wasn't available. I asked about adding it to self-hosted runners and they said they'd add it [to their backlog](https://chat.canonical.com/canonical/pl/7ymgodwiiir9mnc41z36hue1yr) (this was ~3 months ago).
But I see you're already installing it all yourself below which works too.